packages/calling-component-bindings/src/callUtils.ts

```diff
@@ -71,6 +71,11 @@ const memoizedUpdateDisplayName = memoizeFnAll((participantId: string, participa
       ...participant,
       displayName: participant.identifier.phoneNumber
     };
+  } else if (participant.displayName === undefined || participant.displayName === '') {
+    return {
+      ...participant,
+      displayName: 'Unnamed Participant'
```

Comment:
WE should be using a locale string instead of a hard coded string. Consider french do we wan to show "Unnamed Participant"?

Reply:
I like this logic better in the reacct-components layer also as it allows Contoso to decide on what behavior they want if someone doesn't have a display name. If we set a display name here they won't know if someone is actually called Unnamed Participant or if the display name was unset. But it will mean adding to the participant list and to the video gallery (I see the video gallery shows no display name also)

Reply:
Updated :)